.gitignore

```diff
@@ -87,3 +87,6 @@ example/ionic-angular/android/app/capacitor.build.gradle
 
 #yarn pack files
 sentry-capacitor-v*.tgz
+
+#sentry tokens
+*.sentryclirc
```

Comment:
L: Do we have some config files with prefixes?

Reply:
if you run the sourcemap scripts from wizard it automatically adds the token files on the sample projects, so better be safe than leaking personal tokens